app.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+console.log('js file is connected');
+
+// const productImageSectionTag = document.getElementById('allProducts');
+const leftProductImgTag = document.getElementById('leftProductImage');
+const centerProductImgTag = document.getElementById('centerProduct');
+const rightProductImgTag = document.getElementById('rightProduct');
+
+let totalClicks = 0;
+
+// //variables to store products already on the page.
+// let leftProductOnPage = null;
+// let centerProductOnPage = null;
+// let rightProductOnPage = null;
+
+//constructor function
+const ProductPicture = function(name, imageSrc){
```

Comment:
One thing to consider is naming- For example, you'll be creating objects that represent Products, so it may be helpful to name this Product or Item, since the picture is more like a characteristic (or property) of the object we're creating.